libraries/lib-datahandler/src/main/thrift/components.thrift

```diff
@@ -194,6 +196,8 @@ struct Release {
     54: optional COTSDetails cotsDetails,
 
     65: optional set<string> mainLicenseIds,
+    66: optional set<string> mainLicenseNames,
+    67: optional set<License> mainLicenses,
```

Comment:
is this ever used apart from being set in ComponentPortlet.java:532? I can't seem to find any usages